front/app/modules/commercial/impact_tracking/index.ts

```diff
@@ -63,6 +66,13 @@ const trackPageView = async (path: string) => {
   // for the same path. This prevents that.
   if (previousPathTracked === path) return;
 
+  // On first homepage load, if we don't enter the platform on a locale,
+  // we set it immediately. Usually, we set it before the page track event
```

Comment:
Actually, I don't understsand what you mean with "it" here. Could you change that?